lib/bundlex/output.ex

```diff
@@ -5,6 +5,7 @@ defmodule Bundlex.Output do
     Mix.shell().info("Bundlex: " <> msg)
   end
 
+  @spec raise(term()) :: no_return()
```

Comment:
I think we should require `binary()` here instead of `term()`. In other case `<>` will fail.